src/thriftpyi/utils.py

```diff
@@ -17,6 +17,8 @@ def get_python_type(ttype: int, is_required: bool, meta: List) -> str:
         TType.SET: _get_set,
         TType.LIST: _get_list,
     }
+    if TType.BINARY != TType.STRING:
```

Comment:
I think this condition belongs to `_get_binary` method. That way it will be easier to unit test it. Something like this will do:

```python
def get_python_type(...):
    type_map = {
        ...
        TType.BINARY: _get_binary,
        ...
    }

def _get_binary(...):
    if TType.BINARY == TType.STRING:
        return "str"
    return "bytes"
```